tools/loader.md

```diff
@@ -0,0 +1,104 @@
+# Loader instructions
+
+## What is Loader?
+
+Developed by PingCAP, Loader is a data import tool and it imports data to TiDB and MySQL.
+
+[Source code](https://github.com/pingcap/tidb-tools/tree/master/loader)
+
+[Download the Binary](http://download.pingcap.org/tidb-tools-latest-linux-amd64.tar.gz)
+
+## Why did we develop Loader?
+
+Since tools like mysqldump will take us days to migrate massive amounts of data, we used the mydumper/myloader suite of Percona to multi-thread export and import data. During the process, we found that mydumper works well. However, as myloader lacks functions of error retry andd breakpoint transmission, it is inconvenient for us to use. Therefore, we developed loader, which reads the output data files and imports data to TiDB/MySQL through mysql protocol.
```

Comment:
andd -> and
breakpoint transmission -> savepoint
reads the output data files of mydumper